docs/index.adoc

```diff
@@ -157,7 +226,7 @@ The definitions that are specific to the particular implementation types are def
 
 === Structure of the FMU Archive
 
-A2L description files and other files needed for XCP support are placed in `/extra/org.fmi-standard.fmi-ls-xcp`.
+The manifest file, any A2L variable description files and other files needed for XCP support are placed in `/extra/org.fmi-standard.fmi-ls-xcp`.

```

Comment:
instead of any? Either is correct in terms of grammar and meaning, the slight difference in emphasis would not make me care, except that I slightly prefer any in this context.